jekyll/_cci2/webhooks.md

```diff
@@ -35,6 +35,27 @@ Webhooks can be leveraged for various purposes. Some possible examples might inc
 - Trigger internal notification systems to alert people when workflows/jobs complete.
 - Build your own automation plugins and tools.
 
+## Communication Protocol
+{: #communication-protocol }
+
+A webhook is sent whenever an event occurs on the CircleCI platform.
+
+A webhook is sent using an HTTP POST, to the URL that was registered when the webhook was created, with a body encoded using JSON.
+
+CircleCI expects that the server that responds to a webhook will return a 2xx response code. If a non-2xx response is received, CircleCI will retry at a later time. If CircleCI does not receive a response to the webhook within a short period of time, we will assume that delivery has failed, and we will retry at a later time. The timeout period is currently 5 seconds, but is subject to change during the preview period. The exact details of the retry policy are not currently documented, and are subject to change during the preview period. Please get in touch with our team if you have feedback about timeouts and retries.
```

Comment:
Can we put a link to how we want people to get in touch? Support or Discuss or somewhere else?

Reply:
@TylerConlee do you know where this should link to?

Reply:
I've added a link to Canny.